assets/src/components/detours/diversionPage.tsx

```diff
@@ -294,3 +300,18 @@ DiversionPagePanel.Body = DiversionPagePanelBody
 DiversionPage.Panel = DiversionPagePanel
 
 export const Panel = DiversionPagePanel
+
+function descriptors(route: Route, routePattern: RoutePattern) {
```

Comment:
todo: needs a better name

Reply:
also probably shouldn't be `function`

Reply:
> Out of curiosity, how come you wanted this to be extracted as a function, rather than just inlined directly to `DiversionPage` itself?

It was just real messy and I was getting annoyed with how much noise it was adding

Reply:
> And yeah, I _think_ our conventions are that this should be `const descriptors = (...) => {`?

Agreed, though I'm torn because the only reason this can be at the bottom of the file (which is where _I_ would prefer it) is due to hoisting. So converting this to `const` would mean it needs to live at the top of the file, before it's first invocation. So maybe we reconsider this practice at some point but I'll change it for now.

Given it'd have to be the first invocation, do you think in-lining it would be better?

Reply:
Huhhhhhhh - I forgot about that. I think per what our team has been doing so far, it should be a `const` and live at the top of the file (I'm so used to that in our React code that it doesn't really bother me, but now that you mention it, it is weird that we like... sometimes have the "main component" at the top, and sometimes have it at the bottom). Happy to change our rules on that, but we should make an intentional choice to do that, and then follow the new rules.

No objection to in-lining, but no pressure to do so either.

Reply:
@joshlarson what do you think of https://github.com/mbta/skate/compare/4829f3842bf9b7304a55362b17f8c5aa2aa74f80..e180162268a170207ddc2513135b1857d91ef998 ?

Reply:
Probably fine, but since it's above anyway, does it need to be capitalized?

Reply:
oh whoops, didn't mean to do that

Reply:
https://github.com/mbta/skate/compare/b2625564eee19238849772cd11042fdccdff4e81..dca6719b349e19cc09ed41d051d104eed913fbbc